hardhat.config.js

```diff
@@ -44,69 +51,57 @@ const getNetConfig = (networkName, ethAccountName) => {
     ensAddress: netState.ensAddress,
     timeout: 60000,
   }
-  const localhost = {
-    ...base,
-    url: 'http://localhost:8545',
-    chainId: 31337,
-    gas: 80000000, // the same as in Görli
-  }
-  const mainnetfork = {
-    ...base,
-    url: 'http://127.0.0.1:8545',
-    chainId: 1337,
-    gas: 80000000, // the same as in Görli
-  }
   const byNetName = {
-    localhost,
-    mainnetfork,
-    local: {
+    mainnetfork: {
       ...base,
-      accounts: {
-        mnemonic: 'explain tackle mirror kit van hammer degree position ginger unfair soup bonus',
-        count: 30,
-      },
-      url: 'http://localhost:8545',
-      chainId: 1337,
+      url: RPC_URL,
+      chainId: Number(process.env.CHAIN_ID) || 1,
+    },
+    goerlifork: {
+      ...base,
+      url: RPC_URL,
+      chainId: Number(process.env.CHAIN_ID) || 5,
+    },
+    local: {
+      url: RPC_URL,
+      chainId: Number(process.env.CHAIN_ID) || 1337,
     },
     hardhat: {
+      // NB!: forking get enabled if env variable HARDHAT_FORKING_URL is set, see code below
       blockGasLimit: 30000000,
       gasPrice: 0,
       initialBaseFeePerGas: 0,
       allowUnlimitedContractSize: true,
       accounts: {
         // default hardhat's node mnemonic
         mnemonic: 'test test test test test test test test test test test junk',
-        count: 30,
+        count: 10,
```

Comment:
:warning:  I am not sure that it wouldn't break some tests which depend on multiple accounts, actually 
e.g., sanity checker tests if I am not wrong